main.go

```diff
@@ -73,6 +75,7 @@ func main() {
 		clientPrefix = flag.String("client-prefix", "mqtt-benchmark", "MQTT client id prefix (suffixed with '-<client-num>'")
 		clientCert   = flag.String("client-cert", "", "Path to client certificate in PEM format")
 		clientKey    = flag.String("client-key", "", "Path to private clientKey in PEM format")
+		clientCaCert = flag.String("client-cacert", "", "Path to client CA certificate in PEM format")
```

Comment:
Unless I'm missing something, this looks like you want to connect to a server with a TLS cert that is signed by a CA not known to the OS where the tool is run (e.g. self-signed or your private PKI)? 

Sounds to me that the proper name for this parameter would be `serverCACert` ?

Reply:
You're right in that it's for when the CA is not known to the OS, and that this definitely isn't a client CA. I'm not sure I would consider it a broker CA, or just a root CA for both broker/client. Keeping it consistent seems wise though, so I pushed that change up

For what it's worth, here's what Mosquitto uses:
```
--cafile
    Define the path to a file containing PEM encoded CA certificates that are trusted. Used to enable SSL communication.

--cert
    Define the path to a file containing a PEM encoded certificate for this client, if required by the server.

--key
    Define the path to a file containing a PEM encoded private key for this client, if required by the server.
```
https://mosquitto.org/man/mosquitto_pub-1.html

Reply:
Yeah I see your point and now understand why it was initially 'client-ca', they both make sense we just think of it from different ends of the wire :) 

Sometimes it is indeed called just "CA", and that makes sense when both client and server use the same PKI. However, I still believe "broker CA" makes more sense since this tool is a client and it authenticates the broker and needs to know the CA that signed the broker's cert. Thanks!